packages/devextreme/js/__internal/grids/grid_core/keyboard_navigation/scrollable_a11y.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/*
+This extender is to fix accessibilty issue: Scrollable should always have focusable element inside.
+When there are fixed columns on the left and grid has scroll, scrollable element does not have
+any focusable elements inside, because first cell of fixed table gets tabIndex.
+*/
+
+import $, { dxElementWrapper } from '@js/core/renderer';
+import { isDefined, isEmptyObject } from '@js/core/utils/type';
+import eventsEngine from '@js/events/core/events_engine';
+import { ModuleType } from '@ts/grids/grid_core/m_types';
+
+import type { KeyboardNavigationController } from './m_keyboard_navigation';
+
+// eslint-disable-next-line max-len
+export const keyboardNavigationScrollableA11yExtender = (Base: ModuleType<KeyboardNavigationController>): ModuleType<KeyboardNavigationController> => class ScrollableA11yExtender extends Base {
+  private _$firstNotFixedCell: dxElementWrapper | undefined;
+
+  private rowsViewFocusOutHandlerContext!: (event: any) => void;
+
+  init(): void {
+    super.init();
+
+    // eslint-disable-next-line max-len
+    this.rowsViewFocusOutHandlerContext = this.rowsViewFocusOutHandlerContext || this.rowsViewFocusOutHandler.bind(this);
```

Comment:
Minor comment.

Maybe we can replace the ```||``` with ```??``` and split it into two lines (and remove the eslint disable comment)?
```suggestion
    this.rowsViewFocusOutHandlerContext = this.rowsViewFocusOutHandlerContext 
      ?? this.rowsViewFocusOutHandler.bind(this);
```

Reply:
ok